### DISCUSSION TITLE
squeezelite-esp32 as speaker system for Android phone.

### DISCUSSION THREAD

**Discussion by zarss** [sle118/squeezelite-esp32#392] [Q&A]
Pretty sure, I am looking into the wrong direction, therefore cannot find anything. 
Is it possible and how can I use LMS setup to play live audio from my phone? Like, if I watch Youtube video, sound coming from squeezelite players? I can do that with the TV, but cannot find anything usable for LMS.  Thank you.

**Comment by sle118** [comment]
There is a discussion covering the topic here
https://forums.slimdevices.com/forum/user-forums/logitech-media-server/1627865-bluetooth-in

As for our project, it doesn't implement Bluetooth in such way as BT in is limited to local playback
